loader_hub/file/audio/base.py

```diff
@@ -43,13 +59,41 @@ def load_data(
             # Extract audio from video
             audio = video.split_to_mono()[0]
 
-            file_str = str(file)[:-4] + ".mp3"
+            file = str(file)[:-4] + ".mp3"
             # export file
-            audio.export(file_str, format="mp3")
-
-        model = cast(whisper.Whisper, self.parser_config["model"])
-        result = model.transcribe(str(file))
-
-        transcript = result["text"]
-
-        return [Document(transcript, extra_info=extra_info)]
+            audio.export(file, format="mp3")
+
+        transcript = ""
+
+        if self.model_type == "openai_whisper":
+            model = cast(whisper.Whisper, self.parser_config["model"])
+            result = model.transcribe(str(file))
+
+            transcript = result["text"]
+        else:
```

Comment:
similar comment to above, explicitly do `elif self.model_type == "gladia"` and raise ValueError in `else`